engines/wintermute/base/base_file_manager.cpp

```diff
@@ -386,7 +394,7 @@ bool BaseFileManager::hasFile(const Common::String &filename) {
 	if (diskFileExists(filename)) {
 		return true;
 	}
-	if (_packages.hasFile(filename)) {
+	if (_packages.hasFile(backwardSlashesPath)) {
 		return true;    // We don't bother checking if the file can actually be opened, something bigger is wrong if that is the case.
 	}
 	if (!_detectionMode && _resources->hasFile(filename)) {
```

Comment:
This fixes the issue about the paths coming from the scripts. If I understand correctly, only reversing slashes for file paths in packages is consistent with the rest of the code. Do we need any macro guards here?

Reply:
Alpha Polaris might be the first game that happens to combine such queries with backward-slashed filenames. Which would explain why this hasn't been seen before.

BUT:

That doesn't guarantee that we don't have games somehow expecting forward-slashes. 

BUT: As far as I understand we've been unconditionally doing this when opening files, so it should (as far as I know, be safe to do without macro guards.)